test/sys/test_aio.rs

```diff
@@ -26,6 +26,7 @@ fn poll_aio(mut aiocb: &mut AioCb) -> Result<()> {
 // bindings.  So it's sufficient to check that AioCb.cancel returned any
 // AioCancelStat value.
 #[test]
+#[cfg_attr(all(target_env = "musl", target_arch = "x86_64"), ignore)]
```

Comment:
So all of these tests fail in x86_64 musl but pass in i686 musl?  Weird.  Does cross use qemu to run i686 tests?  I would think that it wouldn't need to.